express/src/routes/sign-in.ts

```diff
@@ -19,6 +20,7 @@ import {processLoginOtpMobile} from "../middleware/sign-in-middleware";
 const router = express.Router();
 
 router.get("/sign-in", showSignInFormEmail);
+router.get("/you-need-to-sign-in-again", signInAgain);
```

Comment:
I believe the URL for the timeout page should be `/session-timeout`. Please check against the design spec.

Reply:
Done.